apps/laboratory/tests/shared/fixtures/w3m-wallet-fixture.ts

```diff
@@ -26,4 +26,21 @@ export const testMW = base.extend<ModalWalletFixture>({
     await use(walletValidator)
   }
 })
+export const testMWSiwe = siwe.extend<ModalWalletFixture>({
+  walletPage: async ({ context, browserName }, use) => {
+    // WalletPage needs clipboard permissions with chromium to paste URI
+    if (browserName === 'chromium') {
```

Comment:
Seems like a lot of duplicate code here. Can there be a better abstraction? At least make the clipboard permission thing a function

Reply:
Once `email` comes this will deffo be too much copy/pasta. I sort of deliberately went for copy/pasta here so when `email` comes I can look at the right way of doing this. Other options to look at are polymorphism/inheritance.

Reply:
Agreed; don't want to over-abstract too soon